src/pylogctx/core.py

```diff
@@ -153,7 +164,7 @@ class AddContextFormatter(logging.Formatter):
     def format(self, record):
         msg = super(AddContextFormatter, self).format(record)
         context_str = ' '.join([
-            '{}:{}'.format(k, v) for k, v in context.items()
+            '{}:{}'.format(k, v) for k, v in context.as_dict().items()
```

Comment:
C'est vrai que la fusion récursive de dictionnaire impose de l'implémenter dans `as_dict()` plutôt que dans `items`. Pour autant, `.items() reste pratique comme dans ce cas. Je te propose de ne pas toucher cette ligne :-)

Est-ce qu'on veut que tout les utilisateurs de pylogctx fasse `.as_dict().items()` tout le temps ?

Reply:
Ok why not, je remet